hvplot/converter.py

```diff
@@ -252,26 +252,33 @@ class HoloViewsConverter(object):
                          "bottom_right", "right", "left", "top",
                          "bottom")
 
+    _default_plot_opts = {
+        'logx': False, 'logy': False, 'show_legend': True, 'legend_position': 'right',
+        'show_grid': False, 'responsive': False, 'shared_axes': True}
+
+    _default_cmaps = {
+        'linear': 'kbc_r',
+        'categorical': 'Category10',
+        'cyclic': 'colorwheel',
+        'diverging': 'coolwarm'
+    }
+
     def __init__(self, data, x, y, kind=None, by=None, use_index=True,
                  group_label='Variable', value_label='value',
                  backlog=1000, persist=False, use_dask=False,
                  crs=None, fields={}, groupby=None, dynamic=True,
-                 width=None, height=None, shared_axes=True,
-                 grid=False, legend=True, rot=None, title=None,
-                 xlim=None, ylim=None, clim=None, xticks=None, yticks=None,
-                 logx=False, logy=False, loglog=False, hover=True,
+                 grid=None, legend=None, rot=None, title=None,
+                 xlim=None, ylim=None, clim=None,
+                 logx=None, logy=None, loglog=None, hover=True,
                  subplots=False, label=None, invert=False,
                  stacked=False, colorbar=None, fontsize=None,
-                 colormap=None, datashade=False, rasterize=False,
+                 datashade=False, rasterize=False,
                  row=None, col=None, figsize=None, debug=False,
-                 xaxis=True, yaxis=True, framewise=True, aggregator=None,
-                 projection=None, global_extent=False, geo=False,
-                 precompute=False, flip_xaxis=False, flip_yaxis=False,
+                 framewise=True, aggregator=None,
+                 projection=None, global_extent=None, geo=False,
+                 precompute=False, flip_xaxis=None, flip_yaxis=None,
                  dynspread=False, hover_cols=[], x_sampling=None,
-                 y_sampling=None, project=False, xlabel=None, ylabel=None,
-                 clabel=None, xformatter=None, yformatter=None, tools=[],
-                 padding=None, responsive=False, min_width=None,
-                 min_height=None, max_height=None, max_width=None,
+                 y_sampling=None, project=False, tools=[],
```

Comment:
What is the implication of losing the various constructor arguments (colormap, clabel, etc.)?

Reply:
I don't think there is an implication. The same defaults are still being fed in. It is easier to add `plot_opts` now since you'd just add to the `plotwds` list below. These were never exposed via tab completion, so I dont' think they were really serving anyone.